README.md

```diff
@@ -17,19 +17,144 @@ SPDX-License-Identifier: MIT
 [![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2FStanfordSpezi%2FSpeziML%2Fbadge%3Ftype%3Dplatforms)](https://swiftpackageindex.com/StanfordSpezi/SpeziML)
 
 
-The Spezi machine learning module simplifies the integration to incorporate machine learning-related functionality in their Spezi-based applications.
+# Overview
 
-For more information, please refer to the [API documentation](https://swiftpackageindex.com/StanfordSpezi/SpeziML/documentation).
+The Spezi ML Swift Package includes modules that are helpful to integrate ML-related functionality in your application.
+
+# Spezi Open AI
+
+A module that allows you to interact with GPT-based large language models (LLMs) from OpenAI within your Spezi application.
+
+|![Screenshot displaying the OpenAI API Key Onboarding view from Spezi OpenAI.](Sources/SpeziOpenAI/SpeziOpenAI.docc/Resources/OpenAIAPIKeyOnboardingStep.png#gh-light-mode-only) ![Screenshot displaying the OpenAI API Key onboarding  view.](Sources/SpeziOpenAI/SpeziOpenAI.docc/Resources/OpenAIAPIKeyOnboardingStep~dark.png#gh-dark-mode-only)|![Screenshot displaying the Open AI Model Selection Onboarding Step from Spezi OpenAI.](Sources/SpeziOpenAI/SpeziOpenAI.docc/Resources/OpenAIModelSelectionOnboardingStep.png#gh-light-mode-only) ![Screenshot displaying the Open AI Model Selection Onboarding Step.](Sources/SpeziOpenAI/SpeziOpenAI.docc/Resources/OpenAIModelSelectionOnboardingStep~dark.png#gh-dark-mode-only)|![Screenshot displaying the Chat View from Spezi OpenAI.](Sources/SpeziOpenAI/SpeziOpenAI.docc/Resources/ChatView.png#gh-light-mode-only) ![Screenshot displaying the Chat View.](Sources/SpeziOpenAI/SpeziOpenAI.docc/Resources/ChatView~dark.png#gh-dark-mode-only)|
+|:--:|:--:|:--:|
+|`OpenAIAPIKeyOnboardingStep`|`OpenAIModelSelectionOnboardingStep`|`ChatView`|
```

Comment:
It would be great to see if and how we might be able to scale the images so they are all the same size here while keeping the table format.

Reply:
We can use HTML to scale the images and also define the light vs. dark mode as described here in the Github documentation: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to.

I don't believe there is a method to fix the column widths, which is still an issue if the captions differ too much in length leading to a significant width difference between the columns. I just simplified the captions instead of using their full class names.

Reply:
We can do this in HTML. GitHub now supports the HTML `<picture>` element which will allow us to define both the size of the image and the dark and light mode options. I've also simplified the captions so there is not such a significant difference in length.

Reply:
Very nice, that is great!